pkg/provisioner/ironic/ironic.go

```diff
@@ -1338,7 +1341,7 @@ func (p *ironicProvisioner) PowerOff() (result provisioner.Result, err error) {
 			result.Dirty = true
 			return result, nil
 		}
-		result, err = p.changePower(ironicNode, nodes.PowerOff)
+		result, err = p.changePower(ironicNode, nodes.SoftPowerOff)
```

Comment:
If you use soft power off you need to set a delay. If the host did not power off then, then hard power off. so you need to add a delay and a check on the host state

Reply:
Noted, thanks.